<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GeneVisual</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>699</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="overView">
       <attribute name="title">
        <string>OverView</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Select a file to load</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string> PATH</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Directory for save</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string> PATH</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Ignored</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="projections">
       <attribute name="title">
        <string>Projections</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="15" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="dim1"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>groups</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="17">
         <widget class="GraphProjection" name="graphicsView"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Dimension 0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="groups"/>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="dim0"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Dimension 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lpoint_size">
          <property name="text">
           <string>point size</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QSlider" name="point_size">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>15</height>
           </size>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="showProjection">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_rangeColor">
          <property name="text">
           <string>color for range plot</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QDial" name="dial">
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>230</number>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QComboBox" name="palette_name"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Settings...</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphProjection</class>
   <extends>QGraphicsView</extends>
   <header>src/graph/graphprojection.hpp</header>
   <slots>
    <slot>createProjection()</slot>
    <slot>setHue(int)</slot>
    <slot>setGroup(QString)</slot>
    <slot>setDim0(QString)</slot>
    <slot>setDim1(QString)</slot>
    <slot>setPointSize(int)</slot>
    <slot>setSavePath(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDialogFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>newFilePath(QString)</signal>
   <receiver>label</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveProjectionPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>newDirectoryPath(QString)</signal>
   <receiver>label_4</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDialogDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showProjection</sender>
   <signal>clicked()</signal>
   <receiver>graphicsView</receiver>
   <slot>createProjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>point_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setTextPointSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setRangeColor(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>graphicsView</receiver>
   <slot>setHue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>point_size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>graphicsView</receiver>
   <slot>setPointSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groups</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>graphicsView</receiver>
   <slot>setGroup(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim0</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>graphicsView</receiver>
   <slot>setDim0(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim1</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>graphicsView</receiver>
   <slot>setDim1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>newFilePath(QString)</signal>
  <signal>newDirectoryPath(QString)</signal>
  <slot>openDialogFile()</slot>
  <slot>saveProjectionPlot()</slot>
  <slot>openDialogDirectory()</slot>
  <slot>createProjection()</slot>
  <slot>setRangeColor(int)</slot>
  <slot>setTextPointSize(int)</slot>
 </slots>
</ui>
